### PR TITLE
packs/GoogleDrive: add modify labels command

### PR DESCRIPTION
**Pull request by GalNakash-RecoLabs**
Add a command to modify labels in google drive

## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by GalNakash-RecoLabs** [comment]
PR is ready!

File: Packs/GoogleDrive/Integrations/GoogleDrive/GoogleDrive.py [hunk 1]
```diff
@@ -1196,6 +1207,86 @@ def file_replace_existing_command(client: 'GSuiteClient', args: dict[str, str])
...
+    )
+
+
+def get_labels_command(client: 'GSuiteClient', args: dict[str, str]) -> CommandResults:
+    modify_label_request_res = prepare_get_labels_request(
+        client, args, scopes=COMMAND_SCOPES['MODIFY_LABELS_PERMISSIONS_CRUD'])
+    http_request_params = modify_label_request_res['http_request_params']
+
+    full_url = URLS['DRIVE_LABELS'] + '?' + urllib.parse.urlencode(http_request_params)
+    demisto.log(full_url)
```
Comment (ostolero) [inline Packs/GoogleDrive/Integrations/GoogleDrive/GoogleDrive.py:1275]: We don't really use demisto.log anymore let's change to demisto.info and add something more indicative
```suggestion
    demisto.info(f'full url for get labels is: {full_url}')
```

**Review by ostolero — CHANGES_REQUESTED** [review]
Hi Gal, great work!
Everything is really according to out standard and looks great, just a few points-
1. see my comment regarding adding the logger decorator.
2. There are 2 validation failures in the build, the first one is regarding the use of demisto.log (see comment). the second is regarding the use of demisto.results in main. this is not part of your code but if you can change that line to simply `return` that would be greatly appreciated :)
3. You should add another unit test for the coverage test to pass.
Let me you know if you need any help, I've reached out through slack to schedule a demo :)

**Comment by ostolero** [comment]
Thank you Gal for a great demo! 
During the demo we added proper Human readable outputs, and as discussed we still require outputs for the new commands in the yml. [here](https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/generate_outputs/README.md) is the sdk command that helps achieve this.
when these are finished and all validation pass we can merge :)

**Comment by ostolero** [comment]
The context is approved by @melamedbn , need to fix the last validation failures

**Comment by GalNakash-RecoLabs** [comment]
@ostolero anything from my side?

**Review by ostolero — APPROVED** [review]
All looks great!

**Comment by ostolero** [comment]
Hi Gal ,I approved as all the changes are done :) merging this and when the internal pr passes validations we can merge to master. I'll update over Slack when this is expected to show in the marketplace for the client.

**Pull request merged**